assets/js/sched.js

```diff
@@ -724,221 +724,367 @@ jQuery(document).ready(function ($) {
 
     var lightningTalks = [];
 
-        for (var i = 0; i < allTalks.length; i++) {
-            var talk = allTalks[i];
-
-            if (talk.type === "lightningTalk") {
-                lightningTalks.push(talk);
-                continue;
-            }
+    for (var i = 0; i < allTalks.length; i++) {
```

Comment:
Ignore anything below here. I have Prettier format on save on my VSCode so it's just formatting.